imports/ui/clients/searchClient.js

```diff
@@ -146,6 +150,9 @@ Template.searchClient.events(
     'click .client-search-icon-container': () => {
       $('#search-client-keyword').focus();
     },
+    'click .view_all': () => {
+      Router.go($('.view_all').attr('search-client'));
```

Comment:
instead of doing a global jQuery query you can use event parameter passed to this function already